### ISSUE TITLE
Feature request: Add video license to output dict of Vimeo videos

### ISSUE THREAD

**Issue by zhuyifei1999** [rg3/youtube-dl#8726]
(Related ticket for YouTube: #8725)
There is no mentioning of video license in output dict of Vimeo videos:

```
>>> dl.extract_info('https://vimeo.com/157316669', download=False).keys()
WARNING: Parameter outtmpl is bytes, but should be a unicode string. Put  from __future__ import unicode_literals  at the top of your code file or consider switching to Python 3.x.
[debug] Encodings: locale UTF-8, fs UTF-8, out UTF-8, pref UTF-8
[debug] youtube-dl version 2016.03.01
[debug] Python version 2.7.6 - Linux-3.19.0-49-generic-x86_64-with-Ubuntu-14.04-trusty
[debug] exe versions: avconv 9.18-6, avprobe 9.18-6
[debug] Proxy map: {}
[vimeo] 157316669: Downloading webpage
[vimeo] 157316669: Extracting information
[vimeo] 157316669: Downloading webpage
[vimeo] 157316669: Downloading JSON metadata
[vimeo] 157316669: Downloading m3u8 information
[u'display_id', u'upload_date', u'protocol', u'description', u'extractor', u'requested_subtitles', u'format', u'tbr', u'height', u'like_count', u'uploader', u'duration', u'format_id', u'id', u'subtitles', u'view_count', u'playlist', u'thumbnails', u'fps', u'title', u'url', u'extractor_key', u'http_headers', u'ext', u'uploader_id', u'width', u'comment_count', u'playlist_index', u'webpage_url', u'formats', u'thumbnail', u'webpage_url_basename']
```

In a browser, the license is shown in the popup of "more" link beside the uploader.

Why is this needed? It is usually necessary to know the license in order to comply with the terms (see [downstream bug](https://github.com/Toollabs/video2commons/issues/6)).

**Comment by dstftw** [comment]
Do you have any example URL with non CC licensed videos?

**Comment by zhuyifei1999** [comment]
[This one on front page](https://vimeo.com/154708719) seems to have no license.
There are examples for Creative Commons and Public Domain videos on https://vimeo.com/creativecommons

**Comment by JeanFred** [comment]
I had started implementing this ages ago in https://github.com/JeanFred/youtube-dl/commit/423587d696a8caaee65a2b4edf28729cf1d1d720 ; I have no clue whether this is even remotely rebase-able, but let me know :)

**Comment by legoktm** [comment]
@dstftw fixed this with c38a67bcd5df639b9d7e7faa8685e76446803527. Thanks!
